app-laptop/ibam/ibam-0.5.2-r4.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="6"
```

Comment:
This is a deprecated EAPI!
Please bump it, recommended to EAPI=8 if possible.

(Sorry, but that also will mandate a revbump)

Reply:
thanks @arthurzam I'll just do a revbump to EAPI 8 and fix bugs all-in-one

Reply:
done.